client/src/components/user-auth/Login-Form.jsx

```diff
@@ -1,25 +1,44 @@
 import useInput from './hooks/useInput.jsx';
-// import React, { useContext } from 'react';
-// import { UserContext } from '../global/UserContext';
+import axios from 'axios';
+import { useNavigate } from "react-router-dom";
+import { UserContext } from '../global/UserContext';
+import { useContext } from 'react';
 
 export default function LoginForm() {
-  const username = useInput('');
+  const navigate = useNavigate();
+  const email = useInput('');
   const password = useInput('');
-  // const { user } = useContext(UserContext);
+  const { login, user } = useContext(UserContext);
 
-  const handleSubmit = (event) => {
+
+  const handleSubmit = async (event) => {
     event.preventDefault();
     // TODO: Handle form submission
-    console.log(username.value, password.value)
+    try {
+      var { data }= await axios.post(`${process.env.REACT_APP_API_URL}/auth/login`, {
```

Comment:
I noticed that you're using `REACT_APP_API_URL` here, while some of the others including myself are using `REACT_APP_API_URI` instead. 

This isn't a huge deal since we can earily duplicate the variable/rename it in our dotenv files but just a heads up that on the deployed instance we currently use `URI` (not URL)